frontend/app/src/pages/main/managed-workers/create/components/create-worker-form.tsx

```diff
@@ -699,23 +714,18 @@ export default function CreateWorkerForm({
                         {...field}
                         value={machineType}
                         onValueChange={(value) => {
-                          // get the correct machine type from the value
-                          const machineType = machineTypes.find(
+                          const mt = machineTypes.find(
```

Comment:
I'd be in favor of leaving `machineType` as the name here, WDYT?

Reply:
Not sure why that got updated, but yeah reverting